src/main/webapp/app/course/learning-goals/learning-goal-routing.module.ts

```diff
@@ -0,0 +1,42 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { CreateLearningGoalComponent } from 'app/course/learning-goals/create-learning-goal/create-learning-goal.component';
+import { UserRouteAccessService } from 'app/core/auth/user-route-access-service';
+import { LearningGoalManagementComponent } from 'app/course/learning-goals/learning-goal-management/learning-goal-management.component';
+import { EditLearningGoalComponent } from 'app/course/learning-goals/edit-learning-goal/edit-learning-goal.component';
+
+const routes: Routes = [
+    {
+        path: ':courseId/goals/create',
+        component: CreateLearningGoalComponent,
+        data: {
+            authorities: ['ROLE_ADMIN', 'ROLE_INSTRUCTOR'],
+            pageTitle: 'artemisApp.learningGoal.createLearningGoal.title',
+        },
+        canActivate: [UserRouteAccessService],
```

Comment:
This will need an update to properly display the human readable breadcrumbs since #2474 was merged.